ddl/table.go

```diff
@@ -1118,6 +1118,10 @@ func (w *worker) onSetTableFlashReplica(d *ddlCtx, t *meta.Meta, job *model.Job)
 			Count:          replicaInfo.Count,
 			LocationLabels: replicaInfo.Labels,
 		}
+		// check table tiflash mode, if mode is "", set the normal mode as default mode
+		if tblInfo.TiFlashMode == "" {
+			tblInfo.TiFlashMode = model.TiFlashModeNormal
```

Comment:
There could be existing tables with > 1 TiFlash replicas. In this case, the Table Mode for these tables are kept being `""`. I guess maybe we don't need to update the TiFlashMode here, as `TiFlashMode == ""` will be always possible.